clang/include/clang-c/Dependencies.h

```diff
@@ -222,6 +222,17 @@ clang_experimental_DependencyScannerWorker_getFileDependencies_v1(
     CXDependencyScannerWorker Worker, int argc, const char *const *argv,
     const char *WorkingDirectory, CXModuleDiscoveredCallback *MDC,
     void *Context, CXString *error);
+
+/**
+ * Same as \c clang_experimental_DependencyScannerWorker_getFileDependencies_v1,
+ * but get the dependencies by module name alone.
+ */
+CINDEX_LINKAGE CXFileDependencies *
+clang_experimental_DependencyScannerWorkerByModName_getFileDependencies_v0(
+    CXDependencyScannerWorker Worker, int argc, const char *const *argv,
+    const char *WorkingDirectory, CXModuleDiscoveredCallback *MDC,
+    void *Context, CXString *error, const char *LookedUpModuleName);
```

Comment:
Just `ModuleName` might be clearer. Maybe this should be moved after the `argv` parameter, since those are the main inputs to the function.